pkgs/by-name/dm/dmarc-report-converter/package.nix

```diff
@@ -0,0 +1,29 @@
+{ buildGoModule, fetchFromGitHub, lib, pkgs }:
+
+buildGoModule rec {
+  pname = "dmarc-report-converter";
+  version = "0.6.5";
+
+  src = fetchFromGitHub {
+    owner = "tierpod";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-4rAQhZmqYldilCKomBfuyqS0vcUg5yS4nqp84XSjam4=";
+  };
+
+  vendorHash = null;
```

Comment:
Wouldn't it be better to provide a source hash (as described [here](https://ryantm.github.io/nixpkgs/languages-frameworks/go/))?

Reply:
As far as I can see this doesn't work for this package.
With a broken hash I'm getting this message: `vendor folder exists, please set 'vendorHash = null;' in your expression`. With lib.fakeSha256 I'm getting message: `error: hash '0000000000000000000000000000000000000000000000000000000000000000' does not include a type, nor is the type otherwise known from context`